sleuth/sleuth/enrich_tls.py

```diff
@@ -44,120 +44,58 @@
 VALID = 2
 
 
-security_categories = {
-    "unknown": 0,
-    "avoid": 1,
-    "legacy": 2,
-    "acceptable": 3,
-    "recommended": 4,
-}
+class Policy:
+    'Class to import seclevel policy'
+
+    def __init__(self, filename, failure_threshold):
+        cur_dir = os.path.dirname(__file__)
+        policy_path = os.path.join(cur_dir, filename)
+        with open(policy_path) as f:
+            policy_data = json.load(f)
+            self.classifications = policy_data["classification"]
+            self.rules = policy_data["rules"]
+            self.failure_threshold = int(failure_threshold) if failure_threshold else policy_data["default_failure_threshold"]
+
+    def seclevel(self, x):
+        seclevel_names = {v: k for k, v in self.classifications.iteritems()}
```

Comment:
Do you need to flip-flop the key/values here? You should be able to iterate over the items directly in a loop.